modules/ml/src/main/java/org/apache/ignite/ml/math/primitives/vector/AbstractVector.java

```diff
@@ -160,6 +183,18 @@ protected void checkIndex(int idx) {
         return storageGet(idx);
     }
 
+    /** {@inheritDoc} */
+    @Override public <T extends Serializable> T getRaw(int idx) {
+        checkIndex(idx);
+
+        return sto.getRaw(idx);
+    }
+
+    /** {@inheritDoc} */
+    @Override public <T extends Serializable> T getRawX(int idx) {
```

Comment:
Why somewhere you use `checkIndex(idx)`, but somewhere not?

Reply:
Please read javadoc for get*X methods:
"Gets the value at specified index without checking for index boundaries"